config/webpack.server.js

```diff
@@ -39,6 +40,7 @@ module.exports = (env, argv) => {
 
     plugins: [
       !isProduction && new webpack.HotModuleReplacementPlugin(),
+      !isProduction && new ReactRefreshWebpackPlugin(),
```

Comment:
Does this work?

Reply:
I think so? Any reason why it wouldn't?

Reply:
Haven't looked too much at this but I would think fast refresh needs some we apis to send the updated code? Don't know how it would do this in node? But if it's tested and its documented then that's great!